src/queryResult/singletonStore.ts

```diff
@@ -20,30 +25,57 @@ class QueryResultSingletonStore {
         return QueryResultSingletonStore.instance;
     }
 
-    // Method to set a value in the store
-    public set(key: string, value: any): void {
-        this.store.set(key, value);
+    // Method to set a nested value in the store
```

Comment:
These should all be docstrings instead of regular comments.

```
/**
 * this will show up as a docstring
 */
myFunc() {}

// this will not
myOtherFunc() {}
```